bin/migrate-down

```diff
@@ -43,6 +43,8 @@ if (program.compiler) {
 }
 
 // Setup store
+if (program.store[0] === '.') program.store = path.join(process.cwd(), program.store)
```

Comment:
I think we can skip the `.` check. `path.join` will correctly work with absolute paths, and that would allow just `"migrations/"` to work as well

Reply:
@LinusU I don't think we can. If the designated store is a reference to a module installed under `node_modules`, then resolving the path relative to CWD would prevent the module from being loaded.

Example:
`migrate up --store=some-npm-package`

Reply:
Ah, of corse 👌